package/firmware/linux-firmware/amd.mk

```diff
@@ -14,6 +14,9 @@ define Package/amdgpu-firmware/install
 	$(CP) \
 		$(PKG_BUILD_DIR)/amdgpu/*.bin \
 		$(1)/lib/firmware/amdgpu
+
+	$(INSTALL_DATA) ./files/dcn_3_1_5_dmcub.bin $(1)/lib/firmware/amdgpu
+	$(INSTALL_DATA) ./files/dcn_4_0_1_dmcub.bin $(1)/lib/firmware/amdgpu
```

Comment:
This is not a good idea to include binary files. This will significantly increase the repository size. We can directly download these firmware from Git. An example:
https://github.com/openwrt/openwrt/blob/764576eb5b6bae6c7142bd9e560f8c68da177b0d/package/boot/uboot-mediatek/Makefile#L14

https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/plain/amdgpu/dcn_3_1_5_dmcub.bin?id=9096bad65cb92da2925fe9ad891cbd4961fa6cda
https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/tree/amdgpu/dcn_4_0_1_dmcub.bin?id=9096bad65cb92da2925fe9ad891cbd4961fa6cda

Reply:
Please see latest commit